### PR TITLE
Added string litteral possibility in the tags configuration section

### PR DESCRIPTION
**Pull request by CharlesRaymond1**
Allows to have one or many static string litterals in the pre-existing tags section.

### CODE REVIEW

**Comment by CharlesRaymond1** [comment]
@hynd FYI. Feel free to review.

**Comment by gggiroux** [comment]
looks good to me :+1:

**Comment by mathpl** [comment]
The idea is to provide an hardcoded value for certain tag right?

**Comment by gggiroux** [comment]
right

File: config.go [hunk 1]
```diff
@@ -413,7 +413,12 @@ func LoadConfig(configFile string) Config {
 				switch key {
 				case "tags":
 					for tag, pos := range v {
-						lg.tags[tag.(string)] = pos.(int)
+                                                switch pos.(type) {
+                                                   case int:
+						      lg.tags[tag.(string)] = pos.(int)
+                                                   case string:
+                                                      lg.tags[tag.(string)] = pos.(string)
```
Comment (mathpl) [inline config.go]: add
```
default:
   log.Fatal("Unexpected type for tags section, key %s: %T", tag, post)
```

Comment (CharlesRaymond1) [inline config.go]: :thumbsup:

**Comment by CharlesRaymond1** [comment]
@mathpl Fixed.

**Pull request merged**